### ISSUE TITLE
Binomia - catalogue of collectors?

### ISSUE THREAD

**Issue by rintoult** [AAFC-BICoE/dina-planning#205]
Did y'all already know about this resource? Can we hook into it?

https://bionomia.net/

Heard about it at Botany 2021 conference.

Tara

**Comment by cgendreau** [comment]
Yes we do. You can't really see on the site but if you look on GitHub you will see that it's mostly  @dshorthouse work :)

https://github.com/bionomia/bionomia/graphs/contributors

**Comment by rintoult** [comment]
ah, good to know.

**Comment by dshorthouse** [comment]
O, gawd how do I suppress those commit logs?! Bionomia is a pet project off the side of my desk that I launched before I landed at AAFC.

This business of secondary linking of strings to things (verbatim to authority) has crept into our thinking for DINA, especially with respect to our Agents. We see it in our Collecting Events / Places too. We may eventually enhance our Agent module to include unique identifiers like ORCID, wikidata, and perhaps others. Once we push our data out to GBIF (must do that first!), we can absolutely make use of Bionomia to help us disambiguate people names across and within all our collections. It's a crowd-sourcing environment & our whole community is exploring what collaborative disambiguation and round-tripping of links might look like. Two new terms will be introduced to Darwin Core called `identifiedByID` and `recordedByID` this fall. We're in good shape to play along.
